app/services/db/service_provider_quota_limit/any_sp_over_quota_limit.rb

```diff
@@ -0,0 +1,9 @@
+module Db
+  module ServiceProviderQuotaLimit
+    class AnySpOverQuotaLimit
+      def self.call
+        ::ServiceProviderQuotaLimit.where('percent_full >= 100').any?
```

Comment:
this is going to table scan.... is that ok?

Reply:
Yep.  It's run once daily and the sp list is small at the moment